queries.sql

```diff
@@ -2,9 +2,37 @@
 
 SELECT * FROM animals WHERE name LIKE '%mon';
 SELECT name FROM animals WHERE date_of_birth >= DATE '2016-01-01' AND date_of_birth <= DATE '2019-12-31';
-SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT name FROM animals WHERE neutered = '-1' AND escape_attempts < 3;
 SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > DATE '2022-01-01';
+SAVEPOINT FIRST_SAVE;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO FIRST_SAVE;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE SIGN(weight_kg) < 0;
+COMMIT;
+
+SELECT COUNT(age) FROM users;
+SELECT COUNT(name) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered,SUM(escape_attempts) FROM animals GROUP BY(neutered);
+SELECT MAX(weight_kg), MIN(weight_kg) FROM animals;
```

Comment:
- This query should get the minimum and max weight for each type of animal (GROUP_BY Species). Please fix this 🙏 
- Also please include a query to get `the average number of escape attempts per animal type of those born between 1990 and 2000?`